app/src/main/java/com/paris_2/san3a/presentation/screen/more/components/LogoutItem.kt

```diff
@@ -14,7 +17,11 @@ fun LogoutItem(
     onClickItem: () -> Unit
 ) {
     SettingItem(
-        modifier = modifier,
+        modifier = modifier.clickable(
+            interactionSource = remember { MutableInteractionSource() },
+            indication = null,
+            onClick = {onClickItem()}
```

Comment:
```suggestion
            onClick = onClickItem
```

it's better for readability